drivers/Aqara/aqara-feeder/config.yml

```diff
@@ -1,4 +1,5 @@
 name: 'Aqara Feeder'
 packageKey: 'aqara-feeder'
 permissions:
+  aqara-provisioning: {}
```

Comment:
Unless a driver uses the `get_aqara_secret` API it doesn't need this permission.

Reply:
The `aqara-lock` driver does need the permission though.

Reply:
We found a typo in some definitions: this will need to be `aqara_provisioning` instead of `aqara-provisioning`. Sorry for the confusion.